qemu/tests/cfg/live_snapshot_chain.cfg

```diff
@@ -46,16 +61,47 @@
             image_name_sn18 = "images/sn18"
             image_name_sn19 = "images/sn19"
             image_name_sn20 = "images/sn20"
-            file_create = yes
-            check_base_image_image1 = yes
-        - check:
-            snapshot_chain = "image1 sn1 sn2"
-            image_name_sn1 = "images/sn1"
-            image_name_sn2 = "images/sn2"
-            file_create = yes
-            check_base_image = yes
-        - oops:
-            post_snapshot_cmd = {shell_no_reply:echo c > /proc/sysrq-trigger}
-            check_base_image = yes
-            operate_timeout = 10
-            check_alive_cmd = ""
+            image_name_sn21 = "images/sn21"
```

Comment:
Thanks @vivianQizhu .

Code has been updated, could you help review it again?

Thanks.